src/app/explore/hat-applications.service.ts

```diff
@@ -6,14 +6,18 @@ import { AuthService } from '../core/services/auth.service';
 import { HatApplication, HatApplicationSetup } from './hat-application.interface';
 import { SheFeed } from '../she/she-feed.interface';
 import { User } from '../user/user.interface';
+import { CacheService } from '../core/services/cache.service';
 
 @Injectable()
 export class HatApplicationsService {
   private hatUrl: string;
+  private applicationKey = 'applications';
+  private applicationMaxAge = 20; // in minutes
```

Comment:
Constant private values should `readonly` modifier: https://mariusschulz.com/blog/typescript-2-0-read-only-properties